caravel/bin/caravel

```diff
@@ -125,6 +126,67 @@ def refresh_druid():
             "[" + cluster.cluster_name + "]")
     session.commit()
 
+@manager.option(
+    '-p', '--prefix', default="",
+    help="Sync Table Prefix")
+@manager.option(
+    '-d', '--database', default=None,
+    help="Specifies database (use caravel db name, not real db name)")
+def synctable(prefix, database):
+    ''' Sync DB Table with Caravel Table'''
+    print("")
+
+    if (prefix == "" and database is None) or (database == 'main' and prefix == ""):
+        print("If you not set prefix and db name, some system table may be sync to caravel.")
+        
+        check_status = raw_input("Are you sure do this? (Y/N)").lower().strip()
+        if check_status not in ['y', 'yes']:
+            print("Exit sync table")
```

Comment:
The Line 137 print is for separate synctable output and large console info by caravel cmd use